Clew/Controllers/StateViews/PauseTrackingController.swift

```diff
@@ -0,0 +1,85 @@
+        label.textColor = UIColor.white
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.font = label.font.withSize(20)
+        label.text = mainText
+        label.tag = UIView.mainTextTag
+        view.addSubview(label)
+        
+        enterLandmarkDescriptionButton = UIButton.makeImageButton(view,
+                                                                  alignment: UIConstants.ButtonContainerHorizontalAlignment.left,
+                                                                  appearance: UIConstants.ButtonAppearance.textButton(label: "Describe"),
+                                                                  label: "Enter text to help you remember this landmark")
+        
+        recordVoiceNoteButton = UIButton.makeImageButton(view,
+                                                         alignment: UIConstants.ButtonContainerHorizontalAlignment.right,
+                                                         appearance: UIConstants.ButtonAppearance.textButton(label: "Voice Note"),
+                                                         label: "Record audio to help you remember this landmark")
+        
+        confirmAlignmentButton = UIButton.makeImageButton(view,
+                                                          alignment: UIConstants.ButtonContainerHorizontalAlignment.center,
+                                                          appearance: UIConstants.ButtonAppearance.textButton(label: "Align"),
+                                                          label: "Start \(ViewController.alignmentWaitingPeriod)-second alignment countdown")
+        
+        if let parent: UIViewController = parent {
+            enterLandmarkDescriptionButton.addTarget(parent,
+                                     action: #selector(ViewController.showLandmarkInformationDialog),
+                                     for: .touchUpInside)
+            recordVoiceNoteButton.addTarget(parent,
+                                       action: #selector(ViewController.recordVoiceNote),
+                                       for: .touchUpInside)
+            confirmAlignmentButton.addTarget(parent,
+                                            action: #selector(ViewController.confirmAlignment),
+                                            for: .touchUpInside)
+        }
+        
+        // Do any additional setup after loading the view.
+        view.addSubview(enterLandmarkDescriptionButton)
+        view.addSubview(recordVoiceNoteButton)
+        view.addSubview(confirmAlignmentButton)
+        
```

Comment:
Petty change, I know, but remove that blank line before the next closing brace on line 84.